zombienets/neumann/single-chain.toml

```diff
@@ -0,0 +1,24 @@
+[settings]
+provider = "native"
+timeout = 1000
+
+[relaychain]
+default_command = "../polkadot/target/release/polkadot"
+chain = "rococo-local"
```

Comment:
this should probably be `westend-local` as we need to deploy neumann to purestake alphanet which uses westend